server/game/cards/11-DoR/TheStrengthOfTheMountain.js

```diff
@@ -0,0 +1,27 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheStrengthOfTheMountain extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Defending characters do not bow',
+            condition: () => this.game.isDuringConflict(),
+            effect: 'prevent defending characters from bowing or returning home while defending',
+            gameAction: [
+                ability.actions.cardLastingEffect(context => ({
+                    target: context.player.cardsInPlay.filter(card => card.isDefending()),
+                    effect: ability.effects.doesNotBow()
+                })),
+                ability.actions.cardLastingEffect(context => ({
+                    target: context.player.cardsInPlay.filter(card => card.isDefending()),
+                    effect: ability.effects.cardCannot({
+                        cannot: 'returnHome'
```

Comment:
I think this should be 'sendHome' not 'returnHome'.

You're also missing the qualifier `restricts: 'opponentsCardEffects'` - card as written should allow you to use your own cards to move defenders home.